scrapers/Xtube.yml

```diff
@@ -0,0 +1,30 @@
+name: xtube
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - xtube.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //form/h1/text()
+      Date:
+        selector: //script[contains(text(), "dimension10")]/text()
+        replace:
+          - regex: ^.+'(\d{4})(\d{2})(\d{2})'.+$
+            with: $2 $3, $1
+        parseDate: 01 02, 2006
+      Studio:
+        Name:
+          selector: //title/text()
+          replace:
+            - regex: .+
+              with: xtube
+      Details: //meta[@property="og:description"]/@content
+      Performers:
+        Name: //form[@id="postCommentForm"]/input[@name="contentOwnerId"][not (contains(.,"sponsor_"))]/@value

```

Comment:
the `[not (contains(.,"sponsor_"))]` there doesnt do anything
it shouls be in the `@value`
like this
`//form[@id="postCommentForm"]/input[@name="contentOwnerId"]/@value[not (contains(.,"sponsor_"))]`